admin_ui/src/components/InputField.vue

```diff
@@ -194,14 +198,14 @@ export default Vue.extend({
             return name.toLowerCase().split(" ").join("_")
         },
         setTextareaHeight() {
-            let element = this.$refs.textarea
+            let element: any = this.$refs.textarea
```

Comment:
We might be able to use HTMLElement as the type here

Reply:
Unfortunately `HTMLElement` or `HTMLTextAreaElement` does not work.